### PR TITLE
Fix for Masthead.vue file

### PR DESCRIPTION
**Pull request by hujambo-dunia**
Hello Enis and Dannon,

The bug in the masthead was due to a "bad merge". So, I reverted one commit back for the single `Masthead.vue` file as [this commit](https://github.com/anvilproject/galaxy/pull/18/commits/b261cf5d106b8ffb266fb2d1c7ff3bdde96a42de). Then, I manually added/fixed/omitted the "customizations" or other blockers (like failing frontend tests) in [this commit](https://github.com/anvilproject/galaxy/pull/18/commits/ed19546722f0b9ccb0fa28dbb0c18c819ee10e4d). For the "customizations" commit, I have the following questions: 

1. Enis: Can confirm if font should be "Open Sans" (wasn't set in `blue.scss` file, but may have been missing due to bad merge)
2. Enis: Please confirm the customizations are as expected and loads in docker properly. Customizations used follow [instructions here](https://github.com/galaxyproject/galaxy/commit/11053f868578919ce592fc9d14b2dc6d0904b6b9). 
3. Dannon: Could verify my Vue-3 upgrade for the navGuardModal is accurate? How to QA this?
4. Dannon: Removed [these test scripts](https://github.com/anvilproject/galaxy/pull/18/commits/ed19546722f0b9ccb0fa28dbb0c18c819ee10e4d#diff-ad95d20006cfe38c3b65d06a157f62b52731d24090206774ec08d5d53949ccddL90-L103) from `Masthead.test.js` file, was blocking with errors. Let me know if I should modify for AnVIL.

screenshot_Masthead_showing_2

For future reference, customizations instructions and code found in [this archive commit](https://github.com/galaxyproject/galaxy/commit/11053f868578919ce592fc9d14b2dc6d0904b6b9).

## How to test the changes?
(Select all options that apply)
- [x] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

File: client/src/components/Masthead/Masthead.vue [hunk 1]
```diff
@@ -1,114 +1,212 @@
...
-            if (el.toJSON instanceof Function) {
-                asJson = el.toJSON();
-            } else {
-                asJson = el;
+});
+
+const activeTab = ref(props.initialActiveTab);
+const extensionTabs = ref([]);
+const windowToggle = ref(false);
+const anvilLink = ref("https://anvil.terra.bio");
```
Comment (afgane) [inline client/src/components/Masthead/Masthead.vue]:
```suggestion
const anvilLink = ref("https://anvil.terra.bio/#workspaces");
```

**Comment by afgane** [comment]
> Enis: Can confirm if font should be "Open Sans" (wasn't set in blue.scss file, but may have been missing due to bad merge)

No need for Open Sans. Let's stick with the new Galaxy font for its accessibility benefits.

> Enis: Please confirm the customizations are as expected and loads in docker properly. Customizations used follow [instructions here](https://github.com/galaxyproject/galaxy/commit/11053f868578919ce592fc9d14b2dc6d0904b6b9).

If we could add a small margin to the left of the Galaxy logo so it's not flush with the side of the page that'd look nicer. Other than the one suggestion for URL change, lgtm. Thanks.

**Review by afgane — APPROVED** [review]

**Comment by dannon** [comment]
@hujambo-dunia Instead of manually applying changes, I'd really strongly recommend fixing the merge forward correctly for the minimal diff here.  

This will make future maintenance and merges easier.  If you wanted to do this in two steps I would fix the merge forward accepting the upstream changes as appropriate, reapplying only the minimal changes in the PR I linked you yesterday afterward.

As I mentioned yesterday, I think you can likely use the upstream masthead changes to the logo as-is and drop those customizations now, as well.

**Comment by hujambo-dunia** [comment]
Just to recap: From yesterday's discussions, sounds like @dannon will take a try at fixing the overall bad merge issue for better alignment of future merges.

**Comment by dannon** [comment]
Correct; I updated https://github.com/anvilproject/galaxy/tree/release_23.0 yesterday and it should be good to go.

**Comment by afgane** [comment]
We can close this in favor of the updated `release_23.0` branch. I just tested it and it seems to be working as expected.

Thanks to both of you.

**Pull request closed without merging**